.drone.yml

```diff
@@ -40,7 +43,7 @@ steps:
     commands:
       - docker buildx create --driver docker-container --platform linux/amd64 --name local --use
       - docker login --username "$${DOCKER_USERNAME}" --password "$${DOCKER_PASSWORD}"
-      - bldr pack --progress=plain -o autonomy -f pkg.yaml --push=true --cache-to=autonomy/golang:cache --cache-from=autonomy/golang:cache
+      - bldr pack --progress=plain -o autonomy -f pkg.yaml --push=true --cache-to=autonomy/golang:cache --cache-from=autonomy/kernel:cache
```

Comment:
`--cache-from=autonomy/kernel:cache` should be `--cache-from=autonomy/golang:cache`